lib/cadet/devices/devices.ex

```diff
@@ -208,7 +208,7 @@ defmodule Cadet.Devices do
               },
               300,
               [],
-              ''
+              ~c""
```

Comment:
similarly here, this is unrelated to this PR right? Is it cuz it is failing for some test?

Reply:
Hi, yes it was failing the formatting checks and was added when I ran mix format